rules/windows/command_and_control_iexplore_via_com.toml

```diff
@@ -40,7 +42,7 @@ sequence by host.id, user.id with maxspan = 5s
     "*.office365.com",
     "*.office.com"
     )
-  ]
+  ] with runs=5
```

Comment:
dependent on endgameinc/eql/issues/57

In this case, we can either skip the `runs` or manually repeat the sequence to not be blocked by the EQL update. Also, it will allow these fixes to backport beyond 7.16.

If so, add a comment to return to this with `runs` in the query and we can update all rules which specify needing it

Reply:
Removed it by now as it is not urgent